src/components/Button.jsx

```diff
@@ -0,0 +1,13 @@
+function Button({classNames, buttonType, onClickFunction, children}) {
+    return (
+        <button
+            className={ classNames }
+            type={ buttonType }
+            onClick={ onClickFunction }
+        >
+            { children }
+        </button>
+    );
+}
```

Comment:
Dit is nu een doorgeefluik. Je had net zo goed direct het button element kunnen gebruiken